overlay/etc/init.d/net-online

```diff
@@ -0,0 +1,72 @@
+#!/sbin/openrc-run
+# Copyright (c) 2015 The OpenRC Authors.
+# See the Authors file at the top-level directory of this distribution and
+# https://github.com/OpenRC/openrc/blob/master/AUTHORS
+#
+# This file is part of OpenRC. It is subject to the license terms in
+# the LICENSE file found in the top-level directory of this
+# distribution and at https://github.com/OpenRC/openrc/blob/master/LICENSE
+# This file may not be copied, modified, propagated, or distributed
+# except according to the terms contained in the LICENSE file.
+
+description="Delays until the network is online or a specific timeout"
+
+depend()
+{
+	after modules net
+	need sysfs
+	provide network-online
+	keyword -docker -jail -lxc -openvz -prefix -systemd-nspawn -uml -vserver
+}
+
+get_interfaces()
+{
+	local ifname iftype
+	for ifname in /sys/class/net/*; do
+		[ -h "${ifname}" ] || continue
+		[ -e "/sys/devices/virtual/net/${ifname##*/}" ] && continue
```

Comment:
This right ignores virtual devices such as `lo` and any bridges. This may be a problem longer term?

Reply:
Do you have a link to the original file so I can do a diff

Reply:
https://github.com/OpenRC/openrc/blob/0.39.2/init.d/net-online.in